src/main/java/org/folio/inventory/support/JsonHelper.java

```diff
@@ -17,6 +20,20 @@ public static String getNestedProperty(
       : null;
   }
 
+  public static String getString(JsonObject representation, String propertyName) {
+    if (representation != null) {
+      return representation.getString(propertyName);
+    }
+
+    return null;
+  }
+
+  public static void includeIfPresent(JsonObject target, String propertyName, String value) {
```

Comment:
Why does this method use the name target for the JSON object and the one above use representation?